RAP/rap-faq.qmd

```diff
@@ -213,7 +213,9 @@ Questions learning more about RAP and developing skills.
 
 Absolutely. Currently the best way to do this is to ask other teams to share code directly with you. If we can all make more progress with using Git, then this will make it much easier to share repositories and have code open by default for interested analysts to browse.
 
-We are also starting up a knowledge share series, so keep an eye out for that and make sure to contribute yourselves. We'd also encourage analysts to make more use of teams to post questions and ask about other people in our area doing similar types of analysis, no question is too big or too small!
+We run a regular RAP Knowledge Share series in which we share good practice and examples of work. We record each session so that you can refer back to them at a later date if you want to. Previous recordings can be found on [Sharepoint](https://educationgovuk.sharepoint.com/:f:/r/sites/lvewp00086/SSU%20%20open%20sharing/Reproducible%20Analytical%20Pipeline%20(RAP)%20resources/Knowledge-share%20series). Upcoming sessions are advertised on the DfE analysts network mailing list.
```

Comment:
Could the link text be something like 'RAP knowledge share sharepoint folder' or something? Just thinking out of context sharepoint might not make sense / or easily clash if we put other links to sharepoint also on this page